src/models/gaze16/internal-overview.md

```diff
@@ -0,0 +1,25 @@
+# Gazelle (Internal Overview)
+
+## Bottom components and connectors:
+
+![Internal Components](./img/components-highlighted.jpg)
+
+- Webcam/microphone connector is highlighted in pink
+- Fan connectors are highlighted in orange
+- LCD panel connector is highlighted in red
+- Wireless card is highlighted in blue
+- RAM is highlighted in dark green
+- M.2 SSDs are highlighted in yellow
```

Comment:
Since the slots are like this:
1x M.2 (PCIe NVMe Gen 4)
1x M.2 (PCIe NVMe Gen 3 or SATA)

Should we highlight this differently then?

Reply:
I see that we do that in the Parts & Repair section so maybe copy that here as well?